File: chapter-09-existing-app-companion/completed/android/PocketBrief/shared/WristBrief.kt
package com.example.pocketbrief.shared

import kotlinx.serialization.Serializable

// Book: Chapter 9, p.96.
@Serializable
data class WristBrief(
    val displayName: String,
    val statusText: String,
    val nextAppointmentEpochMs: Long?,
    val alertCount: Int,
    val updatedAtEpochMs: Long,
    val isSensitiveHidden: Boolean
)

File: chapter-09-existing-app-companion/completed/ios/PocketBrief/shared/WristBrief.swift
import Foundation

// Book: Chapter 9, p.97.
struct WristBrief: Codable, Sendable {
    let displayName: String
    let statusText: String
    let nextAppointment: Date?
    let alertCount: Int
    let updatedAt: Date
    let isSensitiveHidden: Bool
}
